### ISSUE TITLE
Login failure 502 bad gateway

### ISSUE THREAD

**Issue by burningimlam** [NginxProxyManager/nginx-proxy-manager#4186]
**Checklist**
- Have you pulled and found the error with `jc21/nginx-proxy-manager:latest` docker image?
  - Yes
- Are you sure you're not using someone else's docker image?
  - Yes 
- Have you searched for similar issues (both open and closed)?
  - Yes

**Describe the bug**

When I installed npm and logged in, 502 bad gateway appeared

I have tried to replace the image version and change the installation location. A problem was found when the npm mirror was started he got stuck at Fetching https://ip-ranges.amazonaws.com/ip-ranges.json
Since the system is now booted normally, please forgive me for not providing a complete error log
**Nginx Proxy Manager Version**
2.10.4，latest

**To Reproduce**

**Expected behavior**

**Screenshots**

**Operating System**

**Additional context**

**Comment by burningimlam** [comment]
https://github.com/NginxProxyManager/nginx-proxy-manager/issues/3965

I executed the command in the link to get the system to start properly

commands：
```shell
NPM_CTR_NAME=nginxproxymanager
docker exec $NPM_CTR_NAME sed -i 's/\.then(internalIpRanges\.fetch)//g' /app/index.js
docker restart $NPM_CTR_NAME
```

**Comment by lybtt** [comment]
> #3965
> 
> I executed the command in the link to get the system to start properly
> 
> commands：
> 
> ```shell
> NPM_CTR_NAME=nginxproxymanager
> docker exec $NPM_CTR_NAME sed -i 's/\.then(internalIpRanges\.fetch)//g' /app/index.js
> docker restart $NPM_CTR_NAME
> ```

Indeed, it seems that the request is being blocked.

**Comment by makeup1122** [comment]
没有完全启动导致的502，我是因为国内访问https://ip-ranges.amazonaws.com/ip-ranges.json困难， 添加DNS 8.8.8.8 解决了

**Comment by ghostlitao** [comment]
> 没有完全启动导致的502，我是因为国内访问[https://ip-ranges.amazonaws.com/ip-ranges.json困难，](https://ip-ranges.amazonaws.com/ip-ranges.json%E5%9B%B0%E9%9A%BE%EF%BC%8C) 添加DNS 8.8.8.8 解决了

Yes , Just because it is not fully started, it will cause 502. I am using the intranet and cannot access the internat. 
After the  
``` Fetching https://ip-ranges.amazonaws.com/ip-ranges.json
app-1  | [12/11/2024] [4:57:23 AM] [IP Ranges] › ✖  error
``` shows Error , every thing works fine .
